hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2056,6 +2116,14 @@ private class Emit[C](
   private def present(pt: PType, c: Code[_]): EmitCode =
     EmitCode(Code._empty, false, PCode(pt, c))
 
+  private def presentI(cb: EmitCodeBuilder, pc: PCode): IEmitCode = {
+    val Lpresent = CodeLabel()
+    cb += Lpresent.goto
+    IEmitCode(CodeLabel(), Lpresent, pc)
+  }
+
+  private def presentI(cb: EmitCodeBuilder, pt: PType, c: Code[_]): IEmitCode = presentI(cb, PCode(pt, c))
```

Comment:
If these were moved inside `emitI`, the `cb` and `pt` arguments could be omitted. Up to you.